### DISCUSSION TITLE
Offering help with improving 6820 v3 support

### DISCUSSION THREAD

**Discussion by HashfoxGmbH** [Freetz-NG/freetz-ng#842] [Allgemein]
I built an image for "6820 v3 - UNTESTED (FREETZ_TYPE_6820_V3=y)" based on tag "ng23080"

I can flash the image to the box via tools/push_firmware and upon the initial flash the box successfully boots and is freetzed with seemingly all functionality intact.

However, it does not survive a box reboot. Once a reboot is triggered from within the freetzed firmware, the box winds up looping in its boot cycle forever.

Re-flashing the same Freetz image via tools/push_firmware again provides for ONE other successful boot, the next reboot again triggers the infinite reboot-cycle.

I want to help with meaningful information to better support this model.

I just need some advice on how to help troubleshoot this issue in a meaningful way!

**Comment by fda77** [comment]
Ive no ideo what you did. How can the firmware with freetz could start when its not flashed? And if it was flashed, from why was the unmodified image was flashed? And who did it? You could chekc the "System" page to see what is in the 2nd partition set
